app/views/contribute.html

```diff
@@ -0,0 +1,224 @@
+        href="{{board}}{{label}}Content"
+        >content</a
+      >&#39; on our project board on on the lists of things we want to do for
+      the kit. You can also add your own.
+    </p>
+    <h3 id="do-quality-assurance-or-accessibility-testing">
+      Do quality assurance or accessibility testing
+    </h3>
+    <p>
+      You can make sure that the code in our kit is robust and accessible by:
+    </p>
+    <ul>
+      <li>
+        running automated and manual tests against WCAG 2.2 AA specifications
+      </li>
+      <li>helping us set up pipelines for automated testing in GitHub</li>
+    </ul>
+    <p>
+      Accessibility issues are labelled &#39;<a
+        href="{{board}}{{label}}Accessibility"
+        >accessibility</a
+      >&#39; on our project board on on the lists of things we want to do for
+      the kit. You can also add your own.
+    </p>
+    <h3
+      id="fix-bugs-or-build-new-features-enhancements-for-the-nhs-prototype-kit"
+    >
+      Fix bugs or build new features (&#39;enhancements&#39;) for the NHS
+      prototype kit
+    </h3>
+    <p>You can help:</p>
+    <ul>
+      <li>fix bugs in the kit</li>
+      <li>
+        add features that have been added to NHS Frontend and the NHS design
+        system but are not in the kit yet
+      </li>
+      <li>
+        add functionality that is not yet in the NHS prototype kit but is in
+        other prototype kits - for example is in the GOV.UK Prototype kit
+      </li>
+      <li>build features that make it easier to use the NHS prototype kit</li>
+    </ul>
+    <p>
+      Existing things we want to do for new features are labelled
+      <a
+        href='{{board}}{{label}}"%3Abug%3A+Bug"'
+        >&#39;bug&#39;</a
+      >
+      or
+      <a
+        href="{{board}}{{label}}%22Enhancement%3A+feature+request%22"
+        >Enhancement : feature request’</a
+      >&#39; on our project board on on the lists of things we want to do for
+      the kit. There are also labels for
+      <a
+        href="{{board}}{{label}}Javascript"
+        >Javascript</a
+      >,
+      <a
+        href="{{board}}{{label}}Nunjucks"
+        >Nunjucks</a
+      >
+      or
+      <a
+        href="{{board}}{{label}}SASS%20%2F%20CSS"
+        >SASS / CSS</a
+      >
+      .
+    </p>
+    <p>You can also add your own bugs or ideas for features.</p>
+    <h3 id="improve-the-contribution-process">
+      Improve the contribution process
+    </h3>
+    <p>
+      We want to make the process of contributing as easy and as accessible as
+      possible. Existing things we want to do to improve the contribution
+      process are labelled
+      <a
+        href="{{board}}{{label}}%22Contribution+process%22"
+        >&#39;Contribution&#39;</a
+      >. You can also add your own.
+    </p>
+    <h2 id="get-started-with-a-contribution">
+      Get started with a contribution
+    </h2>
+    <p>If you are confident with code you can create issues with GitHub.</p>
```

Comment:
I've removed the github thing but also added two links to the separate repos just because otherwise it's not clear that they are separate things. I know we can move things around but otherwise it's not intuitive. This also means that we can remove the github link from the footer